Traceback (most recent call last):
  File "/seamless/seamless/core/transformation.py", line 400, in _execute
    result, logs = await self._execute_local(
  File "/seamless/seamless/core/transformation.py", line 541, in _execute_local
    validate_environment(env)
  File "/seamless/seamless/core/environment.py", line 159, in validate_environment
    raise ValueError("Environment error:\n" + err)
ValueError: Environment error:
Docker:
  Cannot execute code locally: current Docker image name is 'rpbs/seamless', whereas 'nonsense' is required


Status: OK
<Silk: 112 >
